route-rs-afpacket/src/input.rs

```diff
@@ -0,0 +1,70 @@
+use afpacket;
+use futures::{
+    self, ready,
+    task::{Context, Poll},
+};
+use route_rs_runtime::link;
+use std::{mem, pin::Pin};
+
+const MTU: usize = 1500;
+
+#[derive(Default)]
+pub struct AfPacketInputLink {
```

Comment:
Documentation needed for this link.  Especially about how to use it on linux systems, etc.